framework/packages/abstract-client/src/lib.rs

```diff
@@ -27,6 +27,10 @@ pub use publisher::{Publisher, PublisherBuilder};
 pub use service::Service;
 pub use source::AccountSource;
 
+// Re-export abstract testing for test-utils
+#[cfg(feature = "test-utils")]
+pub use abstract_testing;
```

Comment:
Why would we want to do this? The abstract-client is supposed to be used exclusively for integration testing while abstract-testing is a (mostly internal) unit-testing library.

Reply:
Removed